### PR TITLE
Update go-github from v28 to v29

### PR DESCRIPTION
**Pull request by mgrachev**
Fix https://github.com/reviewdog/reviewdog/pull/426

### CODE REVIEW

**Comment by mgrachev** [comment]
@haya14busa I got the error:
```
reviewdog: post failed for golint-github-check: failed to create check: json: cannot unmarshal string into Go struct field App.app.events of type github.Event
```

Could you explain where this error might be?

**Comment by haya14busa** [comment]
fix #1370

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	cloud.google.com/go/datastore v1.0.0
 	github.com/bradleyfalzon/ghinstallation v1.1.0
 	github.com/google/go-cmp v0.4.0
-	github.com/google/go-github/v28 v28.1.1
+	github.com/google/go-github/v29 v29.0.1
```
Comment (haya14busa) [inline go.mod]: Can you update it to v29.0.1?

Comment (mgrachev) [reply go.mod]: Done.

**Pull request merged**